src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,66 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private Pair<K,V>[] valuesArr;
+    private int amountElements;
+
+    public StorageImpl() {
+        this.valuesArr = new Pair[MAX_SIZE];
+    }
+
+    public class Pair<K,V> {
```

Comment:
```suggestion
    private class Pair<K,V> {
```
**Private ones (methods, classes) should be after public ones.**